KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/SolrSearchService.java

```diff
@@ -194,6 +194,28 @@ public void tryConnect(String host, int port) throws KeywordSearchServiceExcepti
         }
     }
 
+    /**
+     * Deletes a data source from Solr for a case.
+     * 
+     * @param dataSourceId the id of the data source to delete.
+     * 
+     * @throws org.sleuthkit.autopsy.keywordsearchservice.KeywordSearchServiceException 
+     */
+    @NbBundle.Messages({
+        "SolrSearchService.deleteDataSource.exceptionMessage.noCurrentSolrCore=DeleteDataSource did not contain a current Solr core so could not delete the Data Source",
+    })
+    @Override
+    public void deleteDataSource(Long dataSourceId) throws KeywordSearchServiceException {
+        try {
+            KeywordSearch.getServer().deleteDataSource(dataSourceId);
+        } catch (NoOpenCoreException ex) {
+            logger.log(Level.WARNING, NbBundle.getMessage(SolrSearchService.class,
+                "SolrSearchService.deleteDataSource.exceptionMessage.noCurrentSolrCore"));
+            throw new KeywordSearchServiceException(NbBundle.getMessage(SolrSearchService.class,
+                    "SolrSearchService.deleteDataSource.exceptionMessage.noCurrentSolrCore"));
+        }
```

Comment:
We don't localize exception messages. The reason, for better or worse, is that the one who ultimately ends up reading them is often one of us... 

We don't need to actually log the exception until we get to the end of its propagation. It can be confusing to have duplicate log entries for the same error.

Reply:
Removed log message

Reply:
Perhaps this PR is missing a commit? The logging is still present, and the the exception message is still in a resource bundle for localization.